_sass/leap/atoms/_typography.sass

```diff
@@ -12,14 +12,15 @@ h1, h2, h3, h4, h5, h6
   line-height: 1.3 !important
   text-rendering: optimizeLegibility
 
-
-
-// text sizes for each header
+// Text sizes for each header
 @for $num from 0 through 6
   h#{$num}
     font-size: text-size($num) !important
 
-
+// Letter spacing for headers 0 - 3
+@for $num from 0 through 3
+  h#{$num}
+    letter-spacing: text-tracking($num) !important
```

Comment:
I know this isn't totally new in Leap but I've had to work around few issues related to these `!important` styles on all headings. Do they need to be `!important`?

It gets harder to keep things looking consistent when we have SASS partials that are used in both the app stylesheet without Leap and the new marketing stylesheet that has it. I've been meaning to fix this on the Track cards on the Courses landing page for example.

Reply:
Yeah, it seems odd to set an `!important` on the base elements; in the utility classes it makes sense. I tested out removing the `!important`, and it didn't appear to break anything. It actually allowed me to remove a line of CSS that overrode the `<h3>` font-size on the Track cards on the Courses page (it used another  `!important` to force the font-size). I think that's what you were referring to?

Reply:
Yep, and I remember having to override it a few times when building the signup page too. Thanks for the change!

I also just realized why the letter spacing was bugging me before! When we use a utility class to override the font size in our headings, we're keeping the original letter spacing of the heading tag. In the landing page heroes this really stands out to me with the smaller heading having more letter spacing than the much larger one.